mantidproject/mantid
# Expose fit parameters in JumpFit UI

Fixes #12734.

To test:
- Open Indirect > Bayes > JumpFit
- Load `Babylon5/Public/DanNixon/data/cf_Result.nxs`
- Fit should be generated automatically and parameters updated in property browser
- Change the fit function

Release notes updates [here](http://www.mantidproject.org/index.php?title=Release_Notes_3_5_Indirect_Inelastic&diff=24279&oldid=24276).
